docs/src/content/howto-transparent.md

```diff
@@ -275,6 +275,9 @@ tproxy_user = "nobody"
 
 rdr pass proto tcp from any to any port $redir_ports -> $tproxy
 pass out route-to (lo0 127.0.0.1) proto tcp from any to any port $redir_ports user { != $tproxy_user }
+
+# End the file with a blank newline /!\
```

Comment:
```suggestion
# End the file with a blank newline
```
I'm not too familiar with pf.conf syntax, but does the backslash at the end cancel out the newline? In either case, I don't want to think about it, so let's just remove it.